<launch>

    <arg name="teleop_mode_absolute"   default="true"/>
    <arg name="sim"   default="true"/>
 
    <include file="$(find relaxed_ik)/launch/load_info_file.launch">
            <arg name="info_file_name" value="ur5_allegro_info.yaml"/>            
    </include>
    
    <!-- <include file="$(find relaxed_ik)/launch/relaxed_ik_python.launch"/> -->
    <node pkg="relaxed_ik" type="relaxed_ik_node.py" name="relaxed_ik_node" output="log"/>

    <node pkg="teleoperation_ur5_allegro_leap" type="relaxed_ik_request_interface.py" name="ur5_2_relaxedIK" output="screen">
        <param name="/ur5_allegro_teleop/ur5/teleop_mode_absolute" type="bool" value="$(arg teleop_mode_absolute)" />
        <param name="simulated" type="bool" value="$(arg sim)" />
    </node>

    <node pkg="teleoperation_ur5_allegro_leap" type="leap_ur5_controller.py" name="leap2ur5_target_gen" output="log">
    </node>
        
</launch>